src/Sarif/Core/StackFrame.cs

```diff
@@ -102,7 +102,7 @@ public override string ToString()
 
             if (this.Location?.PhysicalLocation?.ArtifactLocation?.Uri != null)
             {
-                string fileName = this.Location.PhysicalLocation.ArtifactLocation.Uri.LocalPath;
+                string fileName = this.Location.PhysicalLocation.ArtifactLocation.Uri.OriginalString;
```

Comment:
>OriginalString [](http://example.com/codeflow?start=86&length=14)

Why did we make this change? Where are some nice tests for this? :)

Reply:
this change is together with the other regex change in Stack.cs for aim for improvement in Linux, I revert and create a separated issue for it, revert this as well.